src/main/java/classes/entity/AirPlane.java

```diff
@@ -1,38 +1,35 @@
-package classes;
+package classes.entity;
 
+import Exceptions.NegativeArgumentValueException;
 import interfaces.BookSeat;
 
 public class AirPlane implements BookSeat {
     private String name;
     private String modelNumber;
-    private int maxSeats;
+    private int capacity;
     private int rows;
+    private int cols;
     private int bookedSeats;
     private char[][] seats;
 
-    public AirPlane(String planeName, String model, int rows) {
+    public AirPlane(String planeName, String model, int rows, int cols) {
         this.name = planeName;
         this.modelNumber = model;
+        if (rows < 1 || cols < 1){throw new NegativeArgumentValueException("Airplane contains negative number of rows or cols!");}
```

Comment:
if you are doing action in the same line as if-statement do not use `{}`, still you better do it in 3 lines